pallets/collective-proxy/src/lib.rs

```diff
@@ -40,11 +40,36 @@ mod benchmarking;
 pub mod weights;
 pub use weights::WeightInfo;
 
+/// The parameters under which a particular account has a proxy relationship with some other
+/// account.
+#[derive(
+    Encode,
+    Decode,
+    Clone,
+    Copy,
+    Eq,
+    PartialEq,
+    Ord,
+    PartialOrd,
```

Comment:
How come support for ordering is needed?

Reply:
Yeah. It's the leftovers of my design process when I was considering different options. Removed